README.md

```diff
@@ -1,5 +1,116 @@
+[![Generic badge](https://img.shields.io/badge/discussion-welcome-green.svg)](https://github.com/reearth/reearth/discussions)
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+<!-- TODO: change this link later -->
+  <a href="https://github.com/reearth/reearth">
+    <img src="./public/reearth-logo.svg" alt="Logo" width="300" height="300">
+  </a>
+
+  <p align="center">
+    Re:Earth is a state-of-the-art WebGIS with expandability through plug-ins and the ease of running in a browser.
+    <br />
+    <a  href="https://docs.reearth.io/"><strong>Documents >></strong></a>
+    &emsp;&emsp;&emsp;&emsp;
+    <a href="https://docs.reearth.io/oss"><strong>OSS Webpage >></strong></a>
+    <br />
+    <!-- TODO: here -->
+    <a href="">View Demo</a>
+    ·
+    <a href="https://github.com/reearth/reearth/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/reearth/reearth/discussions">Request Feature</a>
+  </p>
+</p>
+
+Re:Earth consist with 2 repositories. Don't forget to check our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories👍
+
+[![reearth-ui](./public/reearth-ui.png)](https://reearth.io)
+
+<!-- ABOUT THE PROJECT -->
+## Features
+🔌 Higy extensible with plugins system
+
+💻 Super handy with running on the browser
+
+💪 Support standard GIS data formats(CSV, KML, CZML, GeoJSON, shapefile)
+
+📢 Easily make the project public
+
+✨ Freely style the map
+
+
+## Built With
+* [React](https://github.com/facebook/react)
+* [TypeScript](https://github.com/microsoft/TypeScript)
+* [Go](https://github.com/golang/go)
+* [GraphQL](https://github.com/graphql)
+* [MongoDB](https://www.mongodb.com/)
+* [WebAssembly](https://webassembly.org/) + [QuickJS](https://github.com/bellard/quickjs)
+
+
+### Prerequisites
+
+**Front-end**
+* Node.js: Latest version of LTS
+
+**Back-end**
+* Golang +1.16
+* Docker
+
+<!-- USAGE EXAMPLES -->
+## Usage
+
+
+_For more examples, please refer to the [Documentation](https://docs.reearth.io)_
+
+
+
+<!-- ROADMAP -->
+## Roadmap
+
+See the [Roadmap](https://github.com/reearth/reearth/projects/1) for a list of proposed features (and known issues).
+If your have any requests for the features, we are welcome! Just create issues [here](https://github.com/reearth/reearth/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) with `feature request` tag.
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+### Contributing Guide
+<!-- TODO: add link later -->
+Read our [contributing guide]() to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to React.
+### Code of Conduct
+Re:Earth has adopted a Code of Conduct that we expect project participants to adhere to. Please read [the full](./CODE_OF_CONDUCT.md) text so that you can understand what actions will and will not be tolerated.
```

Comment:
```suggestion
Re:Earth has adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](./CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
```